it/RetrieveLocalUnitByKeyAcceptanceSpec.scala

```diff
@@ -1,20 +1,24 @@
 import java.time.Month.MARCH
 
-import fixture.ServerAcceptanceSpec
-import it.fixture.ReadsLocalUnit.localUnitReads
-import org.scalatest.OptionValues
 import play.api.http.HeaderNames.CONTENT_TYPE
-import play.api.http.Status.{ BAD_REQUEST, NOT_FOUND, OK }
+import play.api.http.Status.{BAD_REQUEST, NOT_FOUND, OK}
 import play.mvc.Http.MimeTypes.JSON
+import org.scalatest.OptionValues
+
+import uk.gov.ons.sbr.models.enterprise.{EnterpriseLink, Ern}
+import uk.gov.ons.sbr.models.localunit.{LocalUnit, Lurn}
+import uk.gov.ons.sbr.models.reportingunit.{ReportingUnitLink, Rurn}
+import uk.gov.ons.sbr.models.{Address, Period}
+
+import fixture.ServerAcceptanceSpec
+import it.fixture.ReadsLocalUnit.localUnitReads
 import repository.hbase.localunit.LocalUnitColumns._
 import repository.hbase.localunit.LocalUnitQuery
 import support.WithWireMockHBase
-import uk.gov.ons.sbr.models.Period
-import uk.gov.ons.sbr.models.enterprise.{ EnterpriseLink, Ern }
-import uk.gov.ons.sbr.models.localunit.{ Address, LocalUnit, Lurn }
 
 class RetrieveLocalUnitByKeyAcceptanceSpec extends ServerAcceptanceSpec with WithWireMockHBase with OptionValues {
   private val TargetErn = Ern("1000000012")
+  private val TargetRurn = Rurn("91000000012")
```

Comment:
This test does not search for local units with this RURN - so why is it defined here as TargetRurn?